### DISCUSSION TITLE
Lightbulb Overrides Windows Color Profile?

### DISCUSSION THREAD

**Discussion by djblueshift** [Tyrrrz/LightBulb#266] [General]
As per title, I have a custom color/gamma profile but Lightbulb overpowers this. Confirmed by using QuickGamma, adjusting settings, and seeing the color quickly change over as Lightbulb polls. Obviously, during nighttime this is not an issue as I expect the bluelight filter of Lightbulb to take over. Is there any way to get it to "release" or revert to the default color profile I have set in Windows during the day and when I choose the Disable option Lightbulb at night?

**Comment by Tyrrrz** [comment]
Does disabling LightBulb revert the colors back to your configured profile?

**Reply by djblueshift** [reply]
It does not. I am able to select and apply a color profile (and see the change) when it is disabled, but as soon as I click back into the Lightbulb window or it polls again, it applies the daytime color temp/profile and overrides Windows.

**Reply by Tyrrrz** [reply]
And if you exit LightBulb altogether, does it reset to the correct profile?

**Reply by djblueshift** [reply]
It does not. I have to go into color management and click the profile I want and "Set as default" to apply it again.

**Reply by Tyrrrz** [reply]
Yeah, it looks it won't really be easy then. Windows does not provide a way to layer gamma changes, each application (or Windows feature, in this case) just overrides the current ramp with new values. It means that LightBulb doesn't really know what the "default" gamma to fall back to in this case.

The best attempt it can make is to revert the gamma to whatever it was set before LightBulb was launched, but there may still be a race condition if the color profile is applied after and not before.

**Reply by djblueshift** [reply]
Understood, appreciate you looking into it!
